package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Generate Swagger files automatically for Fastify.",
   "main": "index.js",
   "scripts": {
-    "test": "standard && tap test/*.js"
+    "prepare:swagger-ui": "node var/npm/prepare-swagger-ui",
```

Comment:
can you use `prepublish`  instead? https://docs.npmjs.com/misc/scripts.

Reply:
fixed